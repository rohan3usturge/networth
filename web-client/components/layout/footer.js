const Footer = () => (
  <footer>
    <title>Net Worth Calculator</title>
    <link rel="icon" href="/favicon.ico" />
  </footer>
);

export { Footer };
