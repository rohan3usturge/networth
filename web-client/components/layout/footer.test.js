import { Footer } from "./footer";
import { shallow } from "enzyme";

describe("Footer", () => {
  it("should render properly and match snapshot", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
