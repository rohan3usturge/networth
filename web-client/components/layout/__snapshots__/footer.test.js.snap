// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly and match snapshot 1`] = `<footer />`;
